guides/model_creation/ComplexModels.livemd

```diff
@@ -0,0 +1,84 @@
+# Complex Models
+
+```elixir
+Mix.install([
+  {:axon, path: "."},
```

Comment:
We should point to hex before the release, so that the notebook can be run on it's own. For now `{:axon, github: "elixir-nx/axon"}` is probably best :)